verification/curator-service/ui/src/components/EditCase.test.tsx

```diff
@@ -99,23 +118,23 @@ describe('<EditCase />', () => {
         // expect(screen.getByDisplayValue('2020/01/01')).toBeInTheDocument();
         // query them in tests.
         // expect(screen.getByDisplayValue('Paris')).toBeInTheDocument();
-        //expect(await findByText(/Swedish/)).toBeInTheDocument();
-        //expect(getByText('Severe acute respiratory')).toBeInTheDocument();
+        // expect(await findByText(/Swedish/)).toBeInTheDocument();
+        // expect(getByText('Severe acute respiratory')).toBeInTheDocument();
         // expect(
         //     getByDisplayValue('The reference sequence is identical to MN908947'),
         // ).toBeInTheDocument();
-        //expect(getByText('2.35')).toBeInTheDocument();
-        //expect(getByText('48.85')).toBeInTheDocument();
-        //expect(getByDisplayValue('Hypertension')).toBeInTheDocument();
-        //expect(getByDisplayValue('Plane')).toBeInTheDocument();
+        // expect(getByText('2.35')).toBeInTheDocument();
+        // expect(getByText('48.85')).toBeInTheDocument();
```

Comment:
Do we expect to re-enable these at some point? Otherwise better to delete.

Reply:
I'll remove them, those were disabled a long time ago.